.reek

```diff
@@ -7,3 +7,7 @@ exclude_paths:
 
 UtilityFunction:
   public_methods_only: true
+
+FeatureEnvy:
```

Comment:
sao lại thêm cái này?

Reply:
reek nó có warning  refers to 'user' more than self  ở method gavatar_for nên  thêm vào để ignore nó đi